### ISSUE TITLE
Data Compatibility with Ushahidi v3?

### ISSUE THREAD

**Issue by KrishnaPG** [ushahidi/crisisnet#2]
The data received from this API, can it be directly stored into a Ushahidi instance? Are these two products compatible?

**Comment by rjmackay** [comment]
I don't think so.. we don't have a data plugin for it in Ushahidi. And I don't think the API formats match.

cc @vbhv
